x-pack/plugin/monitoring/src/main/resources/monitoring/watches/elasticsearch_nodes.json

```diff
@@ -7,7 +7,7 @@
       "link": "elasticsearch/nodes",
       "severity": 1999,
       "type": "monitoring",
-      "version_created": 7000099,
+      "version_created": 8000099,
```

Comment:
Could this be replaced when the watch is loaded, similar to watch id? Being hardcoded seems to make this prone to getting out of date.

Reply:
++ , however, I would prefer to address that on a separate PR.

Reply:
Is this version correct though? Won't this PR be backported to 7.x?

Reply:
The version is correct for master. I will have to manually change this in the backport. FWIW, i will need to manually change it with the templated approach too, just in 1 place rather then 6. 

This version is used to determine if the watches need to be updated, so we don't want to set to Version.Current otherwise every version update would update them. (which is i guess isn't that bad)

Reply:
If the intention is to have the version created match when the template was changed, wouldn't that be 7.5.0 (assuming backporting to 7.x right now)? Having this version be different in master and 7.x even though the template is the same seems to defeat the purpose?

Reply:
thanks, I see your point now.  I will close out  #47744 and incorporate it here with the version set to 7.5.0

Reply:
Changes merged in and found at https://github.com/elastic/elasticsearch/pull/45308/commits/9d66e2e5e97dab8c314a163c45f29dbcb529d4f2 and https://github.com/elastic/elasticsearch/pull/45308/commits/fcb5c4086acf243e67d1a16161e1239d751c24c6